app/setup.py

```diff
@@ -145,6 +145,23 @@ def before_request():  # pylint: disable=unused-variable
         request_id = str(uuid4())
         logger.new(request_id=request_id)
 
+        session_cookie_present = False
+        csrf_token_present = False
+        if 'session' in request.cookies:
```

Comment:
```suggestion
        session_cookie_present = 'session' in request.cookies
        csrf_token_present = 'csrf_token' in cookie_session
```

Reply:
👍 I've simplified this and removed these variables.